test/TEST-30-ISCSI/test.sh

```diff
@@ -260,7 +257,6 @@ test_setup() {
         inst ./dhcpd.conf /etc/dhcpd.conf
         inst_multiple /etc/nsswitch.conf /etc/rpc /etc/protocols
         inst /etc/group /etc/group
-
         cp -a /etc/ld.so.conf* "$initdir"/etc
         ldconfig -r "$initdir"
         dracut_kernel_post
```

Comment:
Hmm, you didn't change anything here, besides whitespace

Reply:
At one point it included an include line for the link I can add the whitespace back